responsibleai/responsibleai/managers/causal_manager.py

```diff
@@ -219,6 +219,36 @@ def add(
                        f"got {heterogeneity_model}")
             raise UserConfigValidationException(message)
 
+        # Check treatment_cost is valid
+        if isinstance(treatment_cost, int) and \
```

Comment:
Is the first part of the condition needed if the next condition already checks with `and` if the variable is 0? Trying to check if a list variable == 0 may already return False without needing to check its dtype .. ?

Reply:
I guess it is needed as it means that all non-zero treatment_cost will result in user error.